socorro/analysis/correlations/correlations_app.py

```diff
@@ -73,21 +67,82 @@ class CorrelationsApp(FetchTransformSaveWithSeparateNewCrashSourceApp):
         likely_to_be_changed=True,
     )
 
+    required_config.add_option(
+        'transaction_executor_class',
+        default="socorro.database.transaction_executor."
+        "TransactionExecutorWithInfiniteBackoff",
+        doc='a class that will manage transactions',
+        from_string_converter=class_converter,
+        reference_value_from='resource.postgresql',
+    )
+    required_config.add_option(
+        'database_class',
+        default=(
+            'socorro.external.postgresql.connection_context'
+            '.ConnectionContext'
+        ),
+        doc='the class responsible for connecting to Postgres',
+        from_string_converter=class_converter,
+        reference_value_from='resource.postgresql',
+    )
+
+    required_config.add_option(
+        name='date',
+        doc='Specific date to run this for',
+        default='',
+        from_string_converter=date_with_default_yesterday,
+    )
+
+    required_config.add_option(
+        name='product',
+        doc='Product name',
+        default='Firefox',
+    )
+
+    #--------------------------------------------------------------------------
+    def __init__(self, config, quit_check_callback=None):
+        super(CorrelationsApp, self).__init__(config)
```

Comment:
Why are you not passing down the ``quit_check_callback`` param, or doing something with it? As far as I understand it, it's pretty useful, notably if you want to kill a running app.